src/main/java/frc/robot/Robot.java

```diff
@@ -209,13 +209,15 @@ public void robotInit() {
                 elevator
                     .setExtensionCmd(() -> 0.0)
                     .until(() -> elevator.getExtensionMeters() < 0.05),
+                Commands.waitUntil(() -> controller.y().getAsBoolean()),
                 Commands.parallel(
+                    carriage.runVoltageCmd(-CarriageSubsystem.INDEXING_VOLTAGE).withTimeout(0.25),
                     elevator.setExtensionCmd(() -> ElevatorSubsystem.TRAP_EXTENSION_METERS),
                     Commands.waitUntil(
                             () ->
                                 elevator.getExtensionMeters()
                                     > 0.95 * ElevatorSubsystem.TRAP_EXTENSION_METERS)
```

Comment:
are we going to abstract any of these other ones out?

Reply:
meant to highlight `0.95`

Reply:
i wasnt planning on it since most of the others arent reused in so many places